package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "immutable",
-  "version": "4.0.0-rc.2",
+  "version": "4.0.0-rc.2+olivr70",
```

Comment:
It should not have been shared in the PR. This is just to distinguish my own version, which I npm install using the github url. 

Semver spec #10 allows such build metadata, and also specifies that it should be ignored for determining version precedence.